api-tester/src/main/java/com/revenuecat/apitester/java/GoogleProrationModeAPI.java

```diff
@@ -8,7 +8,7 @@ static void check(final GoogleProrationMode mode) {
         switch (mode) {
             case IMMEDIATE_WITHOUT_PRORATION:
             case IMMEDIATE_WITH_TIME_PRORATION:
-            case DEFERRED:
+            // case DEFERRED:
```

Comment:
Did you mean to leave this commented out?

Reply:
Oh I just read that in the description, never mind.

Reply:
We'll uncomment when we want to re-enabled DEFERRED mode

Reply:
Removed the context. Will open a PR re-adding it so we have it ready when we want to re-enable it